src/components/filters/filters.tsx

```diff
@@ -9,31 +9,31 @@ export const Filters: React.FC = () => {
   };
 
   return (
-    <div className="mx-auto max-w-[1536px]">
-      <h2 className="text-xl font-bold mb-2 ml-0">Новинки</h2>
-      <div className="flex space-x-4">
-        <Button
-          variant="filter"
-          isActive={activeFilter === 'popular'}
-          onClick={() => handleFilterClick('popular')}
-        >
-          По популярности <Icon className={'bi-star ml-1'} />
-        </Button>
-        <Button
-          variant="filter"
-          isActive={activeFilter === 'price-up'}
-          onClick={() => handleFilterClick('price-up')}
-        >
-          По алфавиту <Icon className={'bi-sort-alpha-up ml-1'} />
-        </Button>
-        <Button
-          variant="filter"
-          isActive={activeFilter === 'price-down'}
-          onClick={() => handleFilterClick('price-down')}
-        >
-          По алфавиту <Icon className={'bi-sort-alpha-down ml-1'} />
-        </Button>
+      <div className="mx-auto max-w-[1536px]">
+          <h2 className="text-xl font-bold mt-10 mb-4 ml-0">Новинки</h2>
+          <div className="flex space-x-4 gap-5 mb-8">
+              <Button
+                  variant="filter"
+                  isActive={activeFilter === 'popular'}
+                  onClick={() => handleFilterClick('popular')}
```

Comment:
найдите другой способ передачи информации в хендлер, так чтобы было
```suggestion
                  onClick={handleFilterClick}
```
например присвоить нужной кнопке атрибут `data-...` и потом уже в хендлере его вытащить

Reply:
Это не мой код, я за это не отвечала

Reply:
В реальности это не важно) Если это ваш проект и коммент в вашем ПРе, то как правило и править вам))